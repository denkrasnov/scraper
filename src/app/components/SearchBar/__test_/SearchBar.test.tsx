import React from "react";
import { shallow } from "enzyme";

import SearchBox from "..";

describe("SearchBox", () => {
  it("renders", () => {
    const component = shallow(<SearchBox />);
    expect(component).toMatchSnapshot();
  });
});
