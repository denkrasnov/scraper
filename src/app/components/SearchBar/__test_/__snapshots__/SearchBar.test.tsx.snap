// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBox renders 1`] = `
<Box
  maxWidth="739px"
  width="100%"
>
  <Input
    placeholder="Search"
  />
  <Box
    padding="0 0 0 10px"
  >
    <Button>
      search
    </Button>
  </Box>
</Box>
`;
